ansible/playbooks/do_setup.yml

```
@@ -0,0 +1,242 @@
+---
+# this playbook will set up a kubernetes cluster, a mongodb database, create a spaces instance to run browsertrix cloud
+# It is assumed that you have a digital ocean API in your environment and have ACCESS keys for your Spaces
+# the playbook also uses doctl
+# set your env variables to have `DO_API_TOKEN` `DO_AWS_ACCESS_KEY` and `DO_AWS_SECRET_KEY`
+- name: deploy browsertrix on  digital ocrean
+  hosts: localhost
+  connection: local
+  gather_facts: false
+    #  vars_files:
+    # - "../group_vars/do/vault.yml"
+  vars:
+    - droplet_size: "s-1vcpu-2gb"
```

```suggestion
    - droplet_size: "s-1vcpu-2gb"
    - main_node_size: "s-1vcpu-2gb"
    - crawl_node_size: "c-4"
```
can then use both settings in the create command